### PR TITLE
fix: instancing performance

### PR DESCRIPTION
**Pull request by GBirch33**
fix: #1599

It just needs a test to make sure everything runs as per usual and things are as expected.

### CODE REVIEW

**Review by AlejandroAlvarezMelucciDCL — APPROVED** [review]
LGTM :) just left a small question

File: Explorer/Assets/DCL/InfiniteFloor/Floor.mat [hunk 1]
```diff
@@ -164,4 +164,5 @@ Material:
     - _PlaneClipping: {r: -2.1474836e+9, g: 2.1474836e+9, b: -2.1474836e+9, a: 2.1474836e+9}
     - _SecondaryColor: {r: 0, g: 0, b: 0, a: 1}
     - _SpecColor: {r: 0.2, g: 0.2, b: 0.2, a: 1}
+    - _VerticalClipping: {r: -2.1474836e+9, g: 2.1474836e+9, b: 0, a: 0}
```
[the remark this answers is not in the document]
Comment (GBirch33) [reply Explorer/Assets/DCL/InfiniteFloor/Floor.mat:167]: it's just min/max on the clamping values, so it's effectively the extremities of 32bit float

**Review by anicalbano — APPROVED** [review]
🟢 Working as expected

File: Explorer/Assets/DCL/InfiniteFloor/Floor.mat (hunk 1 above)
[the remark this answers is not in the document]
Comment (AlejandroAlvarezMelucciDCL) [reply Explorer/Assets/DCL/InfiniteFloor/Floor.mat:167]: Oh I see, thanks!

**Pull request merged**